GliaWidgets/Component/Bar/CallButtonBar/CallButtonBar.swift

```diff
@@ -24,26 +24,40 @@ class CallButtonBar: UIView {
     private let style: CallButtonBarStyle
     private let stackView = UIStackView()
     private var buttons = [CallButton]()
-    private var effectView = UIVisualEffectView(effect: UIBlurEffect(style: .prominent))
+    private var effectView = UIVisualEffectView(effect: UIBlurEffect(style: .dark))
     private var bottomSpaceConstraint: NSLayoutConstraint!
-    private var kInsets = UIEdgeInsets(top: 3.0, left: 3.0, bottom: 3.0, right: 3.0)
+    private var leftConstraint: NSLayoutConstraint!
+    private var rightConstraint: NSLayoutConstraint!
+    private var verticalAlignConstrait: NSLayoutConstraint!
+    private var kInsets = UIEdgeInsets(top: 7.0, left: 3.0, bottom: 7.0, right: 3.0)
 
     init(with style: CallButtonBarStyle) {
         self.style = style
-        super.init(frame: .zero)
+        super.init()
         setup()
         layout()
-    }
-
-    required init?(coder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
+        adjustStackConstraints()
     }
 
     override func layoutSubviews() {
         super.layoutSubviews()
         adjustBottomSpacing()
     }
 
+    func adjustStackConstraints() {
+        if currentOrientation.isPortrait {
+            verticalAlignConstrait.autoRemove()
+            stackView.spacing = 2
+            leftConstraint.autoInstall()
+            rightConstraint.autoInstall()
+        } else {
+            leftConstraint.autoRemove()
+            rightConstraint.autoRemove()
+            stackView.spacing = 12
```

Comment:
Do these have some kind of descriptive naming that you think might explain these spacings better?